companion/src/updates/repodatamodels.cpp

```diff
@@ -138,7 +138,7 @@ bool RepoFilteredItemModel::filterAcceptsRow(int sourceRow, const QModelIndex &
   }
 }
 
-const int RepoFilteredItemModel::flags(const int id) const
+const int RepoFilteredItemModel::filterFlags(const int id) const
 {
```

Comment:
I purposely named flags as it does not return the current datamodel m_filterflags. This function returns the processing flags for an entry. Please revert.

Reply:
'flags' clashes with a virtual function in the QSortFilterProxyModel class and generates a warning when compiling. Is there another name you would prefer?